packages/@aws-cdk/aws-ec2/lib/vpc-ref.ts

```diff
@@ -335,3 +361,34 @@ export interface VpcSubnetImportProps {
    */
   subnetId: string;
 }
+
+/**
+ * If the placement strategy is completely "default", reify the defaults so
+ * consuming code doesn't have to reimplement the same analysis every time.
+ *
+ * Returns "private subnets" by default.
```

Comment:
You'll have to propagate that default behavior to the documentation of every place that accepts a `VpcPlacementStrategy`. I have a felling some users will sometimes expect "all subnets" to be the default...

Reply:
I don't think "all subnets" is a viable or expected default.

Furthermore, if a construct wants to change the default from `Private` to `Public`, all they have to add is an `if(undefined)` to their usage of this API.

This method only exists so that in the interplay between `subnetIds()` and `subnets()`, we don't have to do the same `if(undefined)` analysis twice.